spec/helpers_spec.rb

```diff
@@ -867,11 +867,12 @@ def qux; end
         end
       RUBY
 
-      source_code, _source_url = @helpers.method_source_code_and_url(method)
+      source_code, source_url = @helpers.method_source_code_and_url(method)
 
       _(source_code).must_be_nil
       # Unfortunately, _source_url is also nil because RDoc does not provide the
       # source code location in this case.
+      _(source_url).must_be_nil
```

Comment:
The omission of this assertion is intentional.

The comment acknowledges that `source_url` is `nil`, but we wish it was not `nil`.  Therefore, we don't want to assert that it "must be" nil.

Reply:
Okay, I've reverted the commit. 👍